### PR TITLE
Complete typescript template

### PR DESCRIPTION
**Pull request by ndim3d**
## Description 

Completed typescript project template.

## What is changed?

- Components
  - implemented TonConnectPage.
- Dependencies
  - added eslint related packages;
  - added `@tonconnect/ui` package.
- CI/CD:
  - added github workflow config (just copied from [solidjs-template](https://github.com/Telegram-Mini-Apps/solidjs-template/tree/master)).
- Docs
  - updated README (copied from [solidjs-template](https://github.com/Telegram-Mini-Apps/solidjs-template/tree/master) and updated for typescript specific).

## Checks

I did all check locally and inside telegram using [telegram web k](https://web.telegram.org/k) and these mini apps:
- [referece](https://t.me/dim_test_template_bot/dim_test_template_app) (solidjs-template);
- [current](https://t.me/dim_test_dev_bot/dim_test_dev_app) (typesciprt-tempalte, only available while local dev-server is running) ;

- [x] UI is similar with [solidjs-template](https://github.com/Telegram-Mini-Apps/solidjs-template/tree/master);
- [x] HomePage (was implemented, just checked it);
  - [x] UI is similar with reference;
  - [x] All routes navigate correctly.
- [x] InitDataPage (was implemented, just checked it);
  - [x] UI is similar with reference;
  - [x] `@telegram-apps/sdk` is used for getting initData.
- [x] ThemeParamsPage (was implemented, just checked it);
  - [x] UI is similar with reference;
  - [x] `@telegram-apps/sdk` is used for getting themeParams;
  - [x] Displayed theme params are updated on user color theme change.
- [x] LaunchParamsPage (was implemented, just checked it);
  - [x] UI is similar with reference;
  - [x] `@telegram-apps/sdk` is used for getting launchParams.
- [x] TONConnectPage
  - [x] UI is similar with reference;
  - [x] First wallet connect;
  - [x] Connected wallet is restored on repeated navigation;
  - [x] Connected wallet is restored on page refresh and first navigation;
  - [x] Wallet disconnect;

### CODE REVIEW

**Comment by ndim3d** [comment]
image (1)
image (2)

In solidjs-template color values are not updated for some reason. I'll create an issue or commit fix later.

**Comment by ndim3d** [comment]
~~ I noticed that code is not formatted now. I have to format code and be consistent with the rest of the codebase. ~~

Done.

File: README.md [hunk 1]
```diff
@@ -3,19 +3,229 @@
...
+## Run
+
+Although Mini Apps are designed to be opened
+within [Telegram applications](https://docs.telegram-mini-apps.com/platform/about#supported-applications),
+you can still develop and test them outside of Telegram during the development process.
+
+To run the application in the development mode, use the `dev` script:
+
+```bash
+npm run dev
```
Comment (heyqbnk) [inline README.md]:
```bash
pnpm run dev
```

Comment (ndim3d) [reply README.md]: Done

**Pull request merged**